code/game/gamemodes/cult/runes.dm

```diff
@@ -887,7 +886,7 @@ var/list/teleport_other_runes = list()
 				log_game("Construct Shell rune failed - not enough plasteel sheets")
 				return
 
-
+/*
```

Comment:
:-1: to removing this. I see this get used a lot when the cult needs to either go loud or make a last stand.